docs/how-to-guides/troubleshoot-openebs.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Troubleshooting OpenEBS
+weight: 10
+---
+
+## Introduction
+
+This guide enlists various issues with OpenEBS and steps to mitigate those problems.
+
+## Prerequisites
+
+* A Kubernetes cluster accessible via `kubectl`.
+
+* [OpenEBS component](../../configuration-reference/components/openebs-operator) installed.
+
+## ISCSI volume mount issue
+
+If you are running Flatcar version `2605.9.0` and deploy OpenEBS with version `0.5.0` of Lokoctl,
+you might run into volume mount issue. The pod trying to mount the volume continues to stay in
+`ContainerCreating` state. To fix this issue follow the following steps. This volume mount issue is
+fixed in [this PR](https://github.com/kinvolk/lokomotive/pull/1266) in Lokomotive and is release as
+a part of `0.6.0`. The nodes deployed using lokoctl version 0.5.0 or before need to do following
+fix.
+
+### Step 1: Fix the self-hosted Kubelet
+
+> **NOTE**: If any of the following commands fail then you might not be running a self-hosted
+> Kubelet in your cluster.
+
+#### Step 1.1: Update the image
+
+```bash
+kubectl -n kube-system set image ds kubelet kubelet=quay.io/kinvolk/kubelet:v1.19.4
+```
+
+#### Step 1.2: Remove the iscsi mount
+
+```bash
+kubectl -n kube-system edit ds kubelet
+```
+
+Remove the following snippet from the kubelet Daemonset configuration:
+
+```yaml
+- mountPath: /usr/sbin/iscsiadm
+  name: iscsiadm
+```
+
+### Step 2: Fix the iscsid service on hosts
+
+SSH into all the worker nodes that will try to mount the volume created by OpenEBS. Run the
+following set of commands to fix the `iscsid` service.
+
+Acquire root credentials:
+
+```bash
+sudo -i
+```
+
+Create the `iscsid` service dropin and restart the service with new config, copy paste the following commands:
+
+```bash
+mkdir /run/systemd/system/iscsid.service.d
```

Comment:
Is using `/run` here on purpose? If so, why?

Reply:
I found all the other places to create a systemd dropin as read-only. I would like to make it persistent. Is there a more permanent way of doing it?

Reply:
Well, `/run` is definitely not permanent :smile: Put them in `/etc/systemd` of course :)

Reply:
Done. I don't know but I got read-only FS error when doing it in /etc, but now it seems to work. Not sure what went wrong when I tried it for the first time.